organization/app/organization/[id]/view-turfs/[turfId]/page.tsx

```
@@ -0,0 +1,802 @@
+        <p className="text-lg">Turf Not Found</p>
+        <button
+          onClick={() => router.back()}
+          className="mt-4 flex items-center gap-2 text-blue-600 hover:text-blue-800 transition-colors"
+        >
+          <FiArrowLeft /> Go Back
+        </button>
+      </div>
+    );
+  }
+
+  const hasOrgData = turf.organization && turf.organization.location;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)